File: code_examples/3-Structured_Problem_Solving/12-Nested_If_Statements/Python/main.py
#!/usr/bin/env python3
"""
Created by: Mr. Coxall
Created on: Sep 2020
This module uses a nested if statement
"""


def main() -> None:
    """The main() function uses a nested if statement, returns None."""

    # input
    cookies_sold = int(input("Enter the number of boxes of cookies you sold: "))
    print("")

    # process & output
    if cookies_sold >= 20:
        if cookies_sold < 30:
            print("You get a small prize.")
        else:
            print("You get a large prize.")
    else:
        print("No prize.")

    print("\nDone.")


if __name__ == "__main__":
    main()

File: code_examples/3-Structured_Problem_Solving/12-Nested_If_Statements/CSharp/main.cs
/* Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program uses a compound boolean statement
*/

using System;

/*
 * The Program class
*/
class Program {
    static void Main() {
        // this function uses a compound boolean statement

        // input
        Console.Write("Enter term mark (as %): ");
        int termMark = Convert.ToInt32(Console.ReadLine());
        Console.Write("Enter project mark (as %): ");
        int projectMark = Convert.ToInt32(Console.ReadLine());
        Console.WriteLine();

        // process & output
        if (termMark >= 50 && projectMark >= 50) {
            Console.WriteLine("You passed the course.");
        } else {
            Console.WriteLine("You did not pass the course.");
        }

        Console.WriteLine("\nDone.");
    }
}